scss/_patterns_navigation.scss

```diff
@@ -361,23 +371,24 @@ $lightness-threshold: 70;
     }
 
     &:hover {
-      background-color: $color-navigation-background--hover;
+      @extend %navigation-link-hover;
+    }
+
+    &:active {
+      background-color: $color-navigation-background--active;
     }
   }
 
   %navigation-link-selected {
-    @include vf-highlight-bar($color-navigation-highlight, left, true, $sph-inner);
+    @extend %navigation-link-hover;
+
+    @include vf-highlight-bar($color-navigation-highlight, left, true, 0);
 
     @media (min-width: $breakpoint-navigation-threshold) {
-      @include vf-highlight-bar($color-navigation-highlight, bottom, false, $sph-inner);
```

Comment:
It seems that we removed the padding from highlight bar everywhere - can we remove it from mixin as well? It unnecessarily complicates it.